pyproject.toml

```diff
@@ -40,17 +40,19 @@ dependencies = [
 ]
 
 [project.optional-dependencies]
-dev = [
+test = [
     "click",
     "pytest",
     "pytest-cov",
     "pytest-lazy-fixture",
     "pytest-sugar",
     "pytest-timeout",
     "s3fs",
-    "scipy"
+    "scipy",
 ]
-
+minio = [
+    # docker-py,
+    "docker"]
```

Comment:
```suggestion
    "docker",
]
```

Reply:
GH is rendering this diff strangely, the idea here is just to remove the commented-out dependency and bump the closing bracket to a new line.